### DISCUSSION TITLE
[主题商店] 希望增加 tdesign 各产品的主题商店 #316

### DISCUSSION THREAD

**Discussion by 315574925** [Tencent/tdesign#317] [Ideas]
通过web页面快速设计构建自己个性化的主题包，帮助企业更好的管理设计系统，提高产品设计全流程的工作效率。
[参考](https://github.com/Tencent/tdesign/issues/316)

**Comment by xiaosansiji** [comment]
目前官网刚上线了主题配置生成器，跟参考的需求比较类似，现在除了一个默认的官方主题，还有一个针对腾讯云的主题，后面还会陆续加更多业务主题，包括开放大家自行上传定制好的主题包的能力：
![image](https://user-images.githubusercontent.com/7600149/236594530-20902d30-2b34-41b4-a536-dad77834e7d6.png)

固定在官网底部，可以访问体验下

**Comment by 315574925** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。
